client/src/components/AboutUsCard.jsx

```
@@ -18,8 +18,12 @@ function AboutUsCard({
             alt="target icon"
           />
           <div className="card-body">
-            <h5 className="card-title">{heading}</h5>
-            <p className="card-text">{paragraphContent}</p>
+            <h5 className="card-title text-center fw-bold">{heading}</h5>
+            <p
+              className="card-text"
+              style={{ whiteSpace: "pre-line" }}
+              dangerouslySetInnerHTML={{ __html: paragraphContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)